en/flying/basic_flying.md

```diff
@@ -55,55 +55,60 @@ The [Failure Detector](../config/safety.md#failure-detector) will automatically
 
 ### Fixed-wing Takeoff
 
-Automatic takeoff using [Takeoff mode (FW)](../flight_modes_fw/takeoff.md) is highly recommended for fixed-wing vehicles!
-You can either [hand-launch](../flight_modes_fw/takeoff.md#catapult-hand-launch) (default) or use [runway takeoff](../flight_modes_fw/takeoff.md#runway-takeoff) (if supported by hardware and configured).
-
-For runway takeoff:
-
-1. Place the vehicle facing the planned takeoff direction (ideally facing the wind)
-1. Activate [Takeoff mode (FW)](../flight_modes_fw/takeoff.md)
-1. [Arm the vehicle](#arm) using an [RC switch](../config/flight_mode.md) or ground station
-
-   The vehicle will ramp up motors, and fly in the indicated direction, until it reaches the (parameter set) clearance height, then enter Hold mode.
-
-
-For catapult/hand-launch:
+:::note
+Manual takeoffs (as well as landings) are tricky to perform and it is thus recommended to takeoff with the automatic modes instead, especially for inexperienced pilots.
+:::
 
-1. Point the vehicle in the direction you want it to take off, in order to set the course (ideally facing the wind)
-1. Activate [Takeoff mode (FW)](../flight_modes_fw/takeoff.md)
-1. [Arm the vehicle](#arm) using an [RC switch](../config/flight_mode.md) or ground station
-1. Throw/launch the vehicle in the direction you want it to take off.
+For manual takeoffs it is recommended to use [Stabilized mode](../flight_modes/README.md#stabilized_fw), [Acro mode](../flight_modes/README.md#acro_fw) or [Manual mode](../flight_modes/README.md#manual_fw) mode.
+[Position mode](../flight_modes/README.md#position_fw) and [Altitude mode](../flight_modes/README.md#altitude_fw) are also possible to takeoff in, but as the controller in these modes can prioritize airspeed over altitude tracking it is important to accelerate the vehicle sufficiently before bringing them airborne (strong thrust if hand-launched, long runway phase for runway takeoff).

```

Comment:
```suggestion
[Stabilized mode](../flight_modes/README.md#stabilized_fw), [Acro mode](../flight_modes/README.md#acro_fw) or [Manual mode](../flight_modes/README.md#manual_fw) mode are recommended for manual takeoff.
[Position mode](../flight_modes/README.md#position_fw) and [Altitude mode](../flight_modes/README.md#altitude_fw) can also be used, but it is important to accelerate the vehicle sufficiently before bringing them airborne — strong thrust if hand-launched, long runway phase for runway takeoff (this is required because the controller in these modes can prioritize airspeed over altitude tracking).
```